src/algorithms/reco/ElectronReconstruction.h

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2023 Daniel Brandenburg
+
+#pragma once
+
+#include <algorithm>
+#include <cmath>
+#include <vector>
+#include <map>
+
+#include <spdlog/spdlog.h>
+
+#include <edm4eic/vector_utils.h>
+
+// Event Model related classes
+#include "edm4hep/MCParticleCollection.h"
+#include "edm4eic/ReconstructedParticleCollection.h"
+#include "edm4eic/MCRecoParticleAssociationCollection.h"
+#include "edm4eic/MCRecoClusterParticleAssociation.h"
```

Comment:
Collection header includes type itself so this is redundant with the next include.

Reply:
done with bf659cb